pyproject.toml

```diff
@@ -1,21 +1,23 @@
 [tool.poetry]
 name = "desdeo-tools"
-version = "0.2.1"
+version = "0.2.2"
 description = "Generic tools and design language used in the DESDEO framework"
-authors = ["Bhupinder Saini"]
+authors = ["Bhupinder Saini", "Giovanni Misitano"]
 license = "MIT"
 
 [tool.poetry.dependencies]
 python = "^3.7"
 numpy = "^1.17"
 pandas = "^0.25.3"
+matplotlib = "^3.2.1"
```

Comment:
As matplotlib is only used in the example notebooks, I think it should be a dev-dependency. Do you have any reason to keep it as a dependency?

Updating and locking packages already takes too long in desdeo-emo, so I want there to be as few packages as dependencies as possible.

Reply:
Makes sense, moving it to dev.